docker/docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 version: '3.4'
 
 services:
-  vision_project_dev:
```

Comment:
Instead of replacing the dev service, you can add another services with the setup you'd like for the interaction. You can then call `docker-compose` run and point to the service (see [here](https://docs.docker.com/compose/reference/run/)).